package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "speedrun-website",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "nuxt",
+    "build": "nuxt build",
+    "start": "nuxt start",
+    "generate": "nuxt generate",
+    "lint:js": "eslint --ext \".js,.vue\" --ignore-path .gitignore .",
+    "lint": "npm run lint:js",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@nuxtjs/axios": "^5.13.6",
```

Comment:
This is personal preference, but I prefer using "5.13.6" over "^5.13.6", that way we know what is actually installed